src/mscorlib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -7,6 +7,27 @@ namespace System.Globalization
     internal sealed partial class GlobalizationMode
     {
         private const string c_InvariantModeConfigSwitch = "System.Globalization.Invariant";
+        // Linux doesn't support environment variable names including dots
+        private const string c_InvariantModeEnvironmentVariable = "DOTNET_System_Globalization_Invariant";
```

Comment:
The existing `DOTNET_` variables are all upper case: `DOTNET_RUNTIME_SERVICING`, `DOTNET_PACKAGES`, `DOTNET_SKIP_FIRST_TIME_EXPERIENCE`, ... . Do we want to be consistent here? (I am not sure ... just asking.)